src/popup/components/NotificationItem.vue

```diff
@@ -79,6 +80,7 @@ export default {
     address: { type: String, default: '' },
     name: { type: String, default: '' },
     text: { type: String, default: '' },
+    id: { type: Number, default: 0 },
```

Comment:
Is it used anywhere - maybe it could be deleted? Or if there is a plan of using it you should remove default value and make it required as `id` should be unique